tests/ci/integration_test_check.py

```diff
@@ -192,7 +193,14 @@ def main():
     gh = GitHub(get_best_robot_token())
     commit = get_commit(gh, pr_info.sha)
 
-    images = [pull_image(get_docker_image(i)) for i in IMAGES]
+    images = []
+    print(f"CPU arch: [{platform.machine()}]")
+    for image_ in IMAGES:
+        print(image_)
+        if platform.machine().startswith("aarch") and "kerberized-hadoop" in image_:
```

Comment:
It shouldn't work. If the image is omitted, it will still be pulled, so it doesn't matter if we skip it here or not.

Reply:
Besides that, `print` line look like the remainings after the local development